### PR TITLE
Urmila12

### PR DESCRIPTION
**Pull request by urmilagagare**

### CODE REVIEW

**Review by nileshp1122 — APPROVED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by vijaya1811 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

File: urmila/HTML/Assignment1/Q1.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  <head>
+  <title>Assighment 1</title>
+  </head>
+  <body>
+  <h1>HTML</h1>
+  <h3>introduction</h3>
+  <p>HTML is a standard markup language for web page</p>
+  </body>
+  </html>
+
```
Comment (Pawan1005) [inline urmila/HTML/Assignment1/Q1.html]: indentation mistake

File: urmila/HTML/Assignment1/Q3.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  	Notepad or TextEdit
+  </h5>
+  <p>
+  	web pages can be created and modified by using profssional HTML editors.
+  	<br>
+  	However,for learing HTML we recommend a simple text Notepad(PC) or TextEdit(MAC)
+  </p>
+  </body>
+  </html><html>
+  <head>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment1/Q3.html:23]: make sure to use correct tags in file

Comment (Pawan1005) [inline urmila/HTML/Assignment1/Q3.html:23]: indentation mistaks

File: urmila/HTML/Assignment1/Q4.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+<P>
+	I am Urmila Gagare, I am 24 years old.
+	I am from Mumbai.
+    my Qualification BCA in computer.
+</P>
+  </body>
+  </html>
+ 
+  
+
```
Comment (Pawan1005) [inline urmila/HTML/Assignment1/Q4.html:15]: remove extra lines and make sure to make proper indentation in your files

File: urmila/HTML/Assignment2/Q1.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+<h2>Initial public offering</h2>
+<p>
+   <small>
+     Google's initial public offering (IPO) took place five year later, on 19<sup>th</sup> August 2004.
+     <br>At that time Larry Page, Sergey Brin, and Eric Schmid adreed to work together at Google for <del>30</del> <u>20</u> years, until the year 2004.
+   </small>
+</p>
+  <h2 style="text-allign: center; color: green;">Thank You!!</h2>
+</body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment2/Q1.html:30]: add new line at the end of file and same indentation mistake

File: urmila/HTML/Assignment3/Q1.html [hunk 5]
```diff
@@ -0,0 +1,71 @@
+     <!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+     <meta name="viewport" contnt ="width=device-width,initial-scale=1.0">
+    <title>
+    <h1>List</h1>
+    </title>
+    <body>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q1.html:10]: head tag end missing

File: urmila/HTML/Assignment3/Q1.html [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+            ii.SYSTEM-2
+            <ul>
+                <li>
+                    Triveni
+                </li>
+                <li>Kalyani</li>
+            </ul>
+        </dd>
+    </dl>
+</body>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q1.html:71]: ending html tag is missing and indentation is wrong

File: urmila/HTML/Assignment3/Q2.html [hunk 7]
```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+  <head>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q2.html:2]: html tag is missing

File: urmila/HTML/Assignment3/Q2.html [hunk 8]
```diff
@@ -0,0 +1,182 @@
...
+        color: #6fa6a0;
+      }
+      a:link {
+        color:red;
+      }
+      a:visited {
+        color: blue;
+      }
+    </style>
+</style>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q2.html:19]: 2 style closing

File: urmila/HTML/Assignment3/Q2.html [hunk 9]
```diff
@@ -0,0 +1,182 @@
...
+  </head>
+<body>
+  <div style="text-align: center">
+    <h1 style="color: black;">Ambersoft</h1>
+    <br />
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q2.html:24]: no need to give that slash

File: urmila/HTML/Assignment3/Q2.html [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+    <a href="KalyaniA3_2main.html">Home</a> |
+    <a href="KalyaniA3_2mi11.html">MI-MEAN</a> |
+    <a href="KalyaniA3_2contact.html">Contact</a>
+    <br />
+  </div>
+  <br />
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q2.html:30]: remove slash

File: urmila/HTML/Assignment3/Q2.html [hunk 11]
```diff
@@ -0,0 +1,182 @@
...
+    us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us.Contact us .Contact us.Contact
+</div>
+</body>
+</html> <!<!DOCTYPE html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q2.html:58]: what is 2 nd html file in same file

File: urmila/HTML/Assignment3/Q2.html [hunk 12]
```diff
@@ -0,0 +1,182 @@
...
+  <head>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q2.html:59]: 2nd html tag is missing

File: urmila/HTML/Assignment3/Q2.html [hunk 13]
```diff
@@ -0,0 +1,182 @@
...
+            }
+            a:link {
+              color:red;
+            }
+            a:visited {
+              color: blue;
+            }
+      </style>
+  </head>
+</head>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q2.html:77]: 2 head tags

File: urmila/HTML/Assignment3/Q2.html [hunk 14]
```diff
@@ -0,0 +1,182 @@
...
+      is home page subheading.This is home page subheading.This is home page
+      subheading.This is home page subheading.This is home page subheading.This
+      is home page subheading.This is home page subheading.This is home page
+      subheading.This is home page subheading.This is home page subheading.This
+      is home page subheading.This is home page subheading.This is home page
+      subheading.This is home page subheading.
+    </p>
+  </body>
+</html>
+<!DOCTYPE html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q2.html:143]: 3rd html page make 3 different pages

File: urmila/HTML/Assignment3/Q2.html [hunk 15]
```diff
@@ -0,0 +1,182 @@
...
+      <li style="color: black;">niket</li>
+    </ul>
+    <ol>
+      <li style="color: black;">Bipin</li>
+      <li style="color: black;">Priyanka</li>
+      <li style="color: black;">Yashasvi</li>
+    </ol>
+  </body>
+</html>
+
```
Comment (Pawan1005) [inline urmila/HTML/Assignment3/Q2.html:182]: indentation missing

File: urmila/HTML/Assignment4/Q1.html [hunk 16]
```diff
@@ -0,0 +1,63 @@
...
+                </tr>
+                <tr>
+                    <th>Female</th>
+                    <td>1.9</td>
+                    <td>0.003</td>
+                    <td>43%</td>
+                </tr>
+            </table>
+        </body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment4/Q1.html:63]: indentation missing 
add new line at the end of file

File: urmila/HTML/Assignment4/Q2.html [hunk 17]
```diff
@@ -0,0 +1,94 @@
...
+            padding: 15px;
+        }
+        .a{
+            border-top:  10px solid gray;
+        }
+        .adc{
+            border-bottom: 10px solid gray;
+        }
+        .w{
+            border-top: transparent;
```
Comment (Pawan1005) [inline urmila/HTML/Assignment4/Q2.html:27]: give proper names to class

File: urmila/HTML/Assignment4/Q2.html [hunk 18]
```diff
@@ -0,0 +1,94 @@
...
+        </tr>
+        <tr class="adc" >
+        <th>4:30</th>
+        <td>&nbsp;</td>
+        <td>&nbsp;</td>
+        <td>&nbsp;</td>
+        </tr>
+        </body>
+        </table>
+</body>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment4/Q2.html:93]: 2 body tag closing 
make proper indentation

File: urmila/HTML/Assignment4/Q2.html [hunk 19]
```diff
@@ -0,0 +1,94 @@
...
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment4/Q2.html:94]: add new line at the end of file

File: urmila/HTML/Assignment4/Q3.html [hunk 20]
```diff
@@ -0,0 +1,80 @@
...
+            <tr>
+                <td>Wednesday</td>
+                <td class="b">8:00 a.m.</td>
+                <td class="d">12:00 p.m.</td>
+                <td>XSL Formatting Object</td>
+            </tr>
+        </body>
+    </table>
+</body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment4/Q3.html:80]: give proper class names 
make indentation for head and body tag
add new line at the end of file

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
check as per comments

**Pull request merged**